homeassistant/components/unifi/strings.json

```diff
@@ -31,6 +31,7 @@
       },
       "device_tracker": {
         "data": {
+          "clients_to_track": "Select specific clients to track",
```

Comment:
How is this feature different than disabling entities?

Reply:
This allows users to only track the few ones they want and get everything else removed in one easy step.